src/http.js

```diff
@@ -14,6 +14,17 @@ export const self = {
   mergeInQueryOrForm,
 };
 
+export class FetchResponseError extends Error {
+  constructor(response, responseError) {
+    super(response.statusText);
+    this.name = 'FetchResponseError';
```

Comment:
I'm a little bit worried about changing the name from `Error` to `FetchResponseError` . By setting a name to a new one, people that may be asserting the error name for `Error` will suddenly get something different:  `FetchResponseError`. That makes it kind backward incompatible. What do you think?

Reply:
I agree, maybe the name can be changed later in the next major release

Reply:
Sounds like a good compromise.